src/status_im/bots/events.cljs

```diff
@@ -52,25 +55,46 @@
 
 (def ^:private keywordize-vector (partial mapv keyword))
 
-;; TODO(janherich): do something with this horrible triple nested reduce so it's more readable
+(defn- map-values [m f]
```

Comment:
How about intoducing some utils ns for those common methods?

Reply:
We actually have such ns (`status-im.utils.utils`), I will add it there

Reply:
I usually prefer reduce for these things it avoids an intermediary structure

Reply:
@yenda The code there is actually using transducers, not lazy sequences, so there is no intermediary structure :)